applications/OptimizationApplication/tests/test_OptimizationApplication.py

```diff
@@ -118,7 +120,8 @@ def AssembleTestSuites():
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([algorithm_tests.analysis_based_tests.algorithm_gradient_projection.test_gradient_projection.TestGradientProjectionAnalysis]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([algorithm_tests.analysis_based_tests.algorithm_nesterov_accelerated_gradient.test_nestervo_accelerated_gradient_analysis.TestNesterovAcceleratedGradientAnalysis]))
 
-
+    # overall tests
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([archara_shape_opt.test_archara_shape_opt.TestArcharaShapeOpt]))
```

Comment:
please check if you have added your tests.

Reply:
done :)